summarization/extractor_output_preprocessor.py

```diff
@@ -126,10 +126,10 @@ def _strip_numbering_from_title_text(self):
     def _strip_numbering_prefix_from_text(self, text):
         numbered_prefix_list = re.findall('^[0-9]+[:,.,)]*', text)
         if numbered_prefix_list == []:
-            return text
+            return text.lstrip()  # strip leading spaces and tabs
```

Comment:
why not rstrip as well? Or just strip()?

Reply:
updated accordingly!